### PR TITLE
[bors] add bors integration

### PR DESCRIPTION
**Pull request by Saviq**

### CODE REVIEW

**Comment by Saviq** [comment]
bors try

File: bors.toml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+status = ["Snap"]
+delete-merged-branches = true
+block-labels = [ "no-merge" ]
+timeout-sec = 1200
```
Comment (AlanGriffiths) [inline bors.toml:4]: This build takes a long time (over 30min), so is this timeout enough?

**Review by AlanGriffiths — APPROVED** [review]
...let's find out empirically:

bors r+

File: bors.toml (hunk 1 above)
Comment (Saviq) [reply bors.toml:4]: Apparently so :shrug:…

**Pull request merged**